data/catalogs/deltares_data.yml

```diff
@@ -1320,8 +1384,14 @@ merit_hydro_ihu_30sec_v1.0:
     source_license: ODC-By 1.0
     source_url: https://zenodo.org/record/5166932#.YVbxJ5pByUk
     source_doi: 10.5281/zenodo.5166932
-    source_version: 1.0
+    processing_notes: topography/merit_hydro_ihu/README
```

Comment:
Did we really do any processing on IHU files compared to what we downloaded? If so could here briefly describe what?